infra/ansible/playbooks/aggregator.yaml

```diff
@@ -0,0 +1,120 @@
+      file:
+        path: /home/{{ ansible_user }}/repos/{{ service }}
+        state: directory
+        mode: '0755'
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+      loop:
+        - aggregator
+
+    - name: Clone Aligned repository
+      git:
+        repo: https://github.com/yetanotherco/aligned_layer.git
+        dest: /home/{{ ansible_user }}/repos/{{ service }}/aligned_layer
+        version: v0.10.2
+      loop:
+        - aggregator
+
+    - name: Set permissions for cloned repository
+      file:
+        path: /home/{{ ansible_user }}/repos/{{ service }}/aligned_layer
+        mode: '0755'
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+        recurse: yes
+
+    - name: Copy ECDSA keystore to server
+      copy:
+        src: '{{ ecdsa_keystore_path }}'
+        dest: /home/{{ ansible_user }}/.keystores/aggregator.ecdsa
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    - name: Copy BLS keystore to server
+      copy:
+        src: '{{ bls_keystore_path }}'
+        dest: /home/{{ ansible_user }}/.keystores/aggregator.bls
+        owner: '{{ ansible_user }}'
+        group: '{{ ansible_user }}'
+
+    - name: Build aggregator
+      shell: 
+        chdir: /home/{{ ansible_user }}/repos/aggregator/aligned_layer/
+        cmd: /usr/local/go/bin/go build -o /home/{{ ansible_user }}/repos/aggregator/aligned_layer/build/aligned-aggregator /home/{{ ansible_user }}/repos/aggregator/aligned_layer/aggregator/cmd/main.go
+
+    - name: Upload config file for aggregator
+      template:
+        src: config-files/config-aggregator.yaml.j2
+        dest: "/home/{{ ansible_user }}/config/config-aggregator.yaml"
+      vars:
+        aligned_layer_deployment_config_file_path: "{{ lookup('ini', 'aligned_layer_deployment_config_file_path', file='ini/config-aggregator.ini') }}"
+        eigen_layer_deployment_config_file_path: "{{ lookup('ini', 'eigen_layer_deployment_config_file_path', file='ini/config-aggregator.ini') }}"
+        eth_rpc_url: "{{ lookup('ini', 'eth_rpc_url', file='ini/config-aggregator.ini') }}"
+        eth_rpc_url_fallback: "{{ lookup('ini', 'eth_rpc_url_fallback', file='ini/config-aggregator.ini') }}"
+        eth_ws_url: "{{ lookup('ini', 'eth_ws_url', file='ini/config-aggregator.ini') }}"
+        eth_ws_url_fallback: "{{ lookup('ini', 'eth_ws_url_fallback', file='ini/config-aggregator.ini') }}"
+        ecdsa_private_key_store_path: "{{ lookup('ini', 'ecdsa_private_key_store_path', file='ini/config-aggregator.ini') }}"
+        ecdsa_private_key_store_password: "{{ lookup('ini', 'ecdsa_private_key_store_password', file='ini/config-aggregator.ini') }}"
+        bls_private_key_store_path: "{{ lookup('ini', 'bls_private_key_store_path', file='ini/config-aggregator.ini') }}"
+        bls_private_key_store_password: "{{ lookup('ini', 'bls_private_key_store_password', file='ini/config-aggregator.ini') }}"
```

Comment:
Aren't the paths to the keystores set in tasks `Copy ECDSA keystore to server` and `Copy BLS keystore to server`? Can't you hardcode them instead of reading from the `.ini` file?

Reply:
> Aren't the paths to the keystores set in tasks Copy ECDSA keystore to server and Copy BLS keystore to server?

Yes, that task is to copy the contents of those files to the server. 

The lines you selected are to meant to fill a config file, so the `ecdsa_private_key_store_path` and `bls_private_key_store_path` are paths to the file, not the content directly.

Reply:
Ok